src/mcts/search.cc

```diff
@@ -276,6 +281,10 @@ std::vector<std::string> Search::GetVerboseStats(Node* node) const {
 
     oss << std::left << std::setw(5)
         << edge.GetMove(is_black_to_move).as_string();
+    
+    float Q = edge.GetQ(fpu, draw_score, /* logit_q= */ false);
```

Comment:
The M_effect calculation needs to be gated by whether M effect is applied or not by checking the MovesLeftThreshold like is done in search.
Also not sure why you changed Q here to not pass logit_q, that seems wrong.